tests/test_estimators/test_loess.py

```diff
@@ -0,0 +1,120 @@
+import numpy as np
+import pytest
+from sklearn.pipeline import Pipeline
+
+from sklego.linear import LoessRegressor
+
+
+def test_loessregressor_init_normal():
```

Comment:
what is this test testing that isn't tested by other tests?

Reply:
It was included to -only- test that initialization goes as expected, following the philosophy of testing one thing only per test.

 Using the .fit to actually check it was a bit ugly, modified it to just asserting initializing with a valid span works.